MinPlatformPkg/Test/Library/TestPointCheckLib/StandaloneMmCheckMemoryAttribute.c

```diff
@@ -0,0 +1,112 @@
+/** @file
+
+Copyright (c) 2017, Intel Corporation. All rights reserved.<BR>
+Copyright (c) 2022, Microsoft Corporation. All rights reserved.<BR>
+SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+#include <PiMm.h>
+#include <Library/MmServicesTableLib.h>
+#include <Library/TestPointCheckLib.h>
+#include <Library/TestPointLib.h>
+#include <Library/DebugLib.h>
+#include <Library/UefiLib.h>
+#include <Library/PrintLib.h>
+#include <Library/BaseMemoryLib.h>
+#include <Library/MemoryAllocationLib.h>
+#include <Guid/MemoryAttributesTable.h>
+#include <Guid/PiSmmMemoryAttributesTable.h>
+#include <Protocol/LoadedImage.h>
+
+VOID
+TestPointDumpMemoryAttributesTable (
+  IN EFI_MEMORY_ATTRIBUTES_TABLE                     *MemoryAttributesTable
+  );
+
+EFI_STATUS
+TestPointCheckImageMemoryAttribute (
```

Comment:
Ditto, why is this function prototype in this file? I did not see its usage below?

Reply:
Removed.